src/llm_vm/server/main.py

```diff
@@ -29,3 +29,6 @@ def generate_hash(input_string):
     sha256_hash = hashlib.sha256()
     sha256_hash.update(str(input_string).encode('utf-8'))
```

Comment:
why hash functions?

Reply:
vestigial code, I didn't add any of this

Reply:
remove it :)